### ISSUE TITLE
[Basic UI] Frames generate errors with null items

### ISSUE THREAD

**Issue by reyhard** [openhab/openhab-webui#1951]
## Which UI are you reporting an issue for?

- [x] Basic UI
- [ ] HABPanel
- [ ] HABot
- [ ] CometVisu

## The problem

After upgrading from 4.0m2 to 4.0m4 I've noticed that Sitemap frames are producing following error when sitemap is rendered in either PC web browser or Android phone.

`[ui.internal.items.ItemUIRegistryImpl] - Failed to retrieve item during widget rendering, item does not exist: Item 'null' could not be found in the item registry`

Adding some random existing item to frame seems to "solve" this issue
 
`Frame label="label" item=SomeRandomItemThatExist {`

## Expected behavior

No errors in the events.log

## Steps to reproduce

Create sitemap with frames and then try to view it on i.e. phone. 

Following sitemap should generate 3 error lines mentioned above
```
sitemap alarmclock label="Alarms" {
    Frame label="1" item=Thermostat_Automatic_Program  {
        Switch item=AlarmClock_Switch icon="screen" 
    }
    Frame label="2" {
        Text item=Phone_01_AlarmClockDate label="Alarm 1 [%s]" icon="time"
    }
    Frame label="3" {
        Text item=Phone_02_AlarmClockDate label="Alarm 2 [%s]" icon="time"
    }
}
```

**Comment by lolodomo** [comment]
It is probably a regression introduced last Sunday by either openhab/openhab-core#3644 or openhab/openhab-core#3635.
I will have a look.

**Comment by lolodomo** [comment]
Log level was changed from DEBUG to WARN in openhab/openhab-core#3644.
This change has to be reverted.

**Comment by lolodomo** [comment]
Closed by openhab/openhab-core#3686
